modules/nw-route-specific-annotations.adoc

```diff
@@ -41,6 +41,17 @@ and "-". The default is the hashed internal key name for the route. |
 `None`: cookies are restricted to the visited site.
 
 This value is applicable to re-encrypt and edge routes only. For more information, see the link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite[SameSite cookies documentation].|
+
+|`haproxy.router.openshift.io/set-forwarded-headers` | Sets the policy for handling the `Forwarded` and `X-Forwarded-For` HTTP headers per route. The values are:
+
+`append`: appends the header, preserving any existing header. This is the default value.
+
+`replace`: sets the header, removing any existing header.
+
+`if-none`: never sets the header, but preserves any existing header.
+
+`replace`: sets the header if it is not already set.| `ROUTER_SET_FORWARDED_HEADERS`
```

Comment:
Seems a bit odd to have `replace` in here twice with different definitions. I assume it does both, depending on if there is an existing header or not, but would it work to combine them in that case?

Reply:
Yikes! I did something wonky. Fixing now.

Reply:
Fixed!